README.md

```diff
@@ -19,10 +19,42 @@ $ npm install @dxos/random-access-multi-storage
 ```javascript
 import { createStorage } from '@dxos/random-access-multi-storage';
 
-const file = createStorage();
-file.write(0, Buffer('hello'));
+const storage = createStorage('./dir');
+
+const file = storage('test.txt')
+file.write(0, Buffer('hello'), err => {
+  file.read(0, 5, (err, data) => {
+    console.log(data.toString())
+  })
+});
 ```
 
+## API
+
+#### Available storages
+
+- `browser`
+  - `STORAGE_CHROME`: Store files using [random-access-chrome-file](https://github.com/dxos/random-access-chrome-file) and the [Chromium File System API](https://web.dev/native-file-system/). Default in Chrome.
+  - `STORAGE_FIREFOX`: Store files using [random-access-idb-mutable-file](https://github.com/random-access-storage/random-access-idb-mutable-file) and the [Firefox IDBMutableFile](https://developer.mozilla.org/en-US/docs/Web/API/IDBMutableFile). Default in Firefox.
+  - `STORAGE_IDB`: Store files in an IDB database using [random-access-idb](https://github.com/random-access-storage/random-access-idb). Fallback.
+  - `STORAGE_RAM`: [random-access-memory](https://github.com/random-access-storage/random-access-memory).
+- `node`
+  - `STORAGE_NODE`: Store files in the real filesystem [random-access-file](https://github.com/random-access-storage/random-access-file). Default.
+  - `STORAGE_RAM`: [random-access-memory](https://github.com/random-access-storage/random-access-memory).
+
+#### `const storage = createStorage(rootPath, storageType)`
+
+It creates a storage factory to create file storage based on `random-access-storage`.
+
+- `rootPath`: Root path to store the files.
+- `storageType`: Storage type. `Optional`.
```

Comment:
Describe defaults values.

Reply:
Is not enough to be explained in the Available Storage?